views/islandora_batch.views_default.inc

```diff
@@ -477,6 +485,11 @@ function islandora_batch_views_default_views() {
   $handler->display->display_options['fields']['created']['table'] = 'islandora_batch_set';
   $handler->display->display_options['fields']['created']['field'] = 'created';
   $handler->display->display_options['fields']['created']['date_format'] = 'long';
+  /* Field: Islandora Batch: Completed Date */
+  $handler->display->display_options['fields']['completed']['id'] = 'completed';
+  $handler->display->display_options['fields']['completed']['table'] = 'islandora_batch_set';
+  $handler->display->display_options['fields']['completed']['field'] = 'completed';
+  $handler->display->display_options['fields']['completed']['date_format'] = 'long';
```

Comment:
I'm not sure how best to instruct ppl to test this. You essentially have to revert the view to make it read back from the views_default.inc file, but that means if you've made custom changes to that View (how? why? anyway it's possible) you could lose them. So I'm hoping we can communicate this with release notes or something because trying to figure out an update hook that does the right thing in all cases is not possible.